### DISCUSSION TITLE
Add SSL?

### DISCUSSION THREAD

**Discussion by jonp92** [anvilistas/amoni#22] [Ideas]
Would it be possible to specify port for Anvil-app-server to serve HTTP and also to enable HTTPS? Or am I missing something? Would it be possible to read in a .yaml config that is already written for anvil-app-server?

**Comment by meatballs** [comment]
In your app folder, you'll find a `config.yaml` file for your app server.

You can set whatever options you want in there.

**Comment by jonp92** [comment]
Thanks! this is awesome work!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
